recipes/rm.rb

```diff
@@ -29,6 +29,10 @@
 end
 
 
+deps = ""
+if exists_local("ndb", "mysqld") 
+  deps = "mysqld.service"
```

Comment:
Influxdb needs to be up before the resourcemanager is started, otherwise you won't get any metric.